mesonbuild/backend/vs2010backend.py

```diff
@@ -1099,6 +1099,11 @@ def gen_vcxproj(self, target, ofname, guid):
         else:
             ET.SubElement(clconf, 'FavorSizeOrSpeed').text = 'Speed'
         # Note: SuppressStartupBanner is /NOLOGO and is 'true' by default
+        if self.name in('vs2017', 'vs2019'):
+            if 'cpp' in file_args:
+                optargs = [x for x in file_args['cpp'] if x.startswith('/std:c++')]
+                if len(optargs)>0:
```

Comment:
using `len()` to check container emptiness is a python anti-pattern, as it o(N):
```python
if optargs:
    ...
```

Reply:
done